docker.local/Dockerfile

```diff
@@ -2,35 +2,34 @@ FROM golang:1.11.4-alpine3.8 as zdns_build
 
 RUN apk add --update --no-cache build-base linux-headers git cmake bash perl grep
 
-RUN apk add gmp gmp-dev openssl-dev && \
-    cd /tmp && \
-    wget -O - https://github.com/herumi/mcl/archive/master.tar.gz | tar xz && \
-    mv mcl* mcl && \
-    make -C mcl -j $(nproc) lib/libmclbn256.so install && \
-    cp mcl/lib/libmclbn256.so /usr/local/lib && \
-    rm -R /tmp/mcl
-
-RUN git clone https://github.com/herumi/bls /tmp/bls && \
-    cd /tmp/bls && \
-    git submodule init && \
-    git submodule update && \
-    make -j $(nproc) install && \
-    cd - && \
-    rm -R /tmp/bls
+# RUN apk add gmp gmp-dev openssl-dev && \
+#     cd /tmp && \
+#     wget -O - https://github.com/herumi/mcl/archive/master.tar.gz | tar xz && \
+#     mv mcl* mcl && \
+#     make -C mcl -j $(nproc) lib/libmclbn256.so install && \
+#     cp mcl/lib/libmclbn256.so /usr/local/lib && \
+#     rm -R /tmp/mcl
+
+# RUN git clone https://github.com/herumi/bls /tmp/bls && \
+#     cd /tmp/bls && \
+#     git submodule init && \
+#     git submodule update && \
+#     make -j $(nproc) install && \
+#     cd - && \
+#     rm -R /tmp/bls
```

Comment:
Remove commented-out code. It's still under source control if we want it back.

Reply:
done